docs/src/model_setup/grids.md

```diff
@@ -139,20 +139,22 @@ grid = RectilinearGrid(size = (Nx, Ny, Nz),
                               z = chebychev_spaced_z_faces)
 ```
 
-We can easily visualize the spacing of ``y`` and ``z`` directions.
+We can easily visualize the spacings of ``y`` and ``z`` directions. We can use, e.g.,
+[`ynodes`](@ref) and [`yspacings`](@ref) to extract the positions and spacings of the
+nodes from the grid.
 
 ```@example 1
 using CairoMakie
 
 fig = Figure(size=(800, 900))
 
 ax1 = Axis(fig[1, 1]; xlabel = "y (m)", ylabel = "y-spacing (m)", limits = (nothing, (0, 250)))
-lines!(ax1, grid.yᵃᶜᵃ[1:Ny], grid.Δyᵃᶜᵃ[1:Ny])
-scatter!(ax1, grid.yᵃᶜᵃ[1:Ny], grid.Δyᵃᶜᵃ[1:Ny])
+lines!(ax1, ynodes(grid, Center()), yspacings(grid, Center()))
+scatter!(ax1, ynodes(grid, Center()), yspacings(grid, Center()))
 
 ax2 = Axis(fig[2, 1]; xlabel = "z-spacing (m)", ylabel = "z (m)", limits = ((0, 50), nothing))
-lines!(ax2, grid.Δzᵃᵃᶜ[1:Nz], grid.zᵃᵃᶜ[1:Nz])
-scatter!(ax2, grid.Δzᵃᵃᶜ[1:Nz], grid.zᵃᵃᶜ[1:Nz])
+lines!(ax2, zspacings(grid, Center()), znodes(grid, Center()))
+scatter!(ax2, zspacings(grid, Center()), znodes(grid, Center()))
```

Comment:
Probably using more lines for all this stuff will make it easier to read

Reply:
good point! done